README.md

```diff
@@ -107,6 +107,9 @@ tenant:
   # regardless of the response from upstream. This can lose metrics if Cortex/Mimir is
   # throwing rejections.
   accept_all: false
+
+  # Optional hard-coded prefix for all tenant values, added with "-" delimiter.
+  tenant_prefix: foobar
```

Comment:
Let's maybe name it just `prefix`? Since we're anyway already in `tenant:` namespace.

Reply:
Fixed. And I changed the code a bit to add some unit-tests for checking headers